flang/lib/Optimizer/Transforms/ExternalABIConversion.cpp

```diff
@@ -0,0 +1,184 @@
+//===- ExternalABIConversion.cpp -- convert name with external ABI ------*-===//
```

Comment:
'-*-' leftover in the emac header ?

Reply:
Just a copy-paste from another file. We might want to make a pass on all files in this directory (an maybe others) because the headers are not consistent.